AElf.OS/Network/INetworkManager.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AElf.Common;
+using AElf.Kernel;
+
+namespace AElf.OS.Network
+{
+    public interface INetworkManager
+    {
+        Task<bool> AddPeerAsync(string address);
+        Task<bool> RemovePeerAsync(string address);
+        List<string> GetPeers();
+        
+        Task<IBlock> GetBlockByHash(Hash hash, string peer = null);
+        Task BroadcastAnnounce(Block b);
```

Comment:
do not broadcast a block, just only send the announce of {block hash, height} or block header

Reply:
Yes agreed, no need for the block at this point.